resources/config/livewire-powergrid.php

```diff
@@ -79,6 +79,19 @@
 
     'filter' => 'inline',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Persisting
+    |--------------------------------------------------------------------------
+    |
+    | PowerGrid supports persisting of the filters, columns and sorting.
+    | 'true': persist in the session.
+    | 'false': persist with cookies (default).
+    |
+    */
+
+    'persist_in_session' => false,
```

Comment:
What do you think of `'persist_driver' => 'cookies' // session`? It is easier to implement if there is another

Reply:
good idea, no problem

Reply:
Its changed. I will write the test tomorrow for the full trait. Whats the name of the right folder in test/ for the test-file?

Reply:
I've added todo tests in https://github.com/Power-Components/livewire-powergrid/pull/1509/commits/76aaa26ada39cab8cd7c349cfdbe7718baef7c21

Reply:
done

can you tell me please, why I cant see the code coverage in the trait? I tried it with: "./vendor/pestphp/pest/bin/pest --coverage tests/Feature/PersistTest.php"